manifest.json

```diff
@@ -0,0 +1,41 @@
+[{
```

Comment:
❓ Do you mind explaining what this file is for, please?

Reply:
The manifest.json contains all the MSGraph and AADGraph permissions required as part of the application registration process. There are alternative ways of building these but this file can be used also from within the Azure Console to set these permissions rather than having the users have to find permissions through the permission selector.